recipes/prep_env.rb

```diff
@@ -73,6 +73,9 @@
 
 include_recipe "aws-parallelcluster::_setup_python"
 
+# Configure additional Networking Interfaces (if present)
+include_recipe "aws-parallelcluster::network_interfaces_config"
```

Comment:
What about executing it after the cloudwatch_agent_configuration?

In this way we'll have the logs of the network setup in cloudwatch.

Reply:
My concern here is that cloudwatch needs the Network to be properly configured to work.